tests/postgres_tests/test_aggregates.py

```diff
@@ -313,6 +313,45 @@ def test_array_agg_lookups(self):
         )
         self.assertCountEqual(qs.get(), [1, 2])
 
+    def test_array_agg_filter_index(self):
+        aggr1 = AggregateTestModel.objects.create(integer_field=1)
+        aggr2 = AggregateTestModel.objects.create(integer_field=2)
+        StatTestModel.objects.create(related_field=aggr1, int1=1, int2=0)
+        StatTestModel.objects.create(related_field=aggr1, int1=2, int2=1)
+        StatTestModel.objects.create(related_field=aggr2, int1=3, int2=0)
+        StatTestModel.objects.create(related_field=aggr2, int1=4, int2=1)
```

Comment:
These could use `bulk_create` to speed up the fixture setup process

```suggestion
        StatTestModel.objects.bulk_create([
            StatTestModel(related_field=aggr1, int1=1, int2=0)
            StatTestModel(related_field=aggr1, int1=2, int2=1)
            StatTestModel(related_field=aggr2, int1=3, int2=0)
            StatTestModel(related_field=aggr2, int1=4, int2=1)
        ])
```